src/styles/utils/_placeholders.scss

```diff
@@ -0,0 +1,19 @@
+%properties-little {
+  border-radius: 10px;
+  animation: rotate 60s linear infinite;
+  background-color: blueviolet;
+  z-index: 2;
+}
+
+%properties-big {
+  border-radius: 10px;
+  animation: rotate 3600s linear infinite;
+  background-color: rgb(95, 17, 17);
+  z-index: 1;
+}
+
+%properties-center {
+  border-radius: 50%;
+  background-color: blueviolet;
+  z-index: 3;
+}
```

Comment:
do you really need to create placeholders for these?

Reply:
to tell the truth ,no